build/tools/packaging/debian_config.json

```diff
@@ -1,17 +1,17 @@
 {
-    "maintainer_name": "Microsoft",
-    "maintainer_email": "nugetaspnet@microsoft.com",
+    "maintainer_name": "MAINTAINER_NAME",
```

Comment:
Consider renaming this file to something other than `.json` (it's not a .json file). In other repos, we use `.in` as  way to indicate when a file is a template. e.g. `debian_config.json.in`.